### ISSUE TITLE
Bug: Android back button doesn't work in post note window 

### ISSUE THREAD

**Issue by bitcoinuser** [lovvtide/satellite-web#15]
Hi,

Bug: Android back button doesn't work in post note window 

![Screenshot_2023-10-15-12-55-43-321_com nostr universe](https://github.com/lovvtide/satellite-web/assets/1178856/08482cb6-d1d3-4ed7-b5f0-53e24d0840a8)

**Comment by lovvtide** [comment]
I don't have an Android phone so I can't test this myself — does the "Cancel" button work at least?

**Comment by bitcoinuser** [comment]
Yes, cancel button works ok. But Android back button do not close the window, instead make the window behind the post window goes to previous window.

**Comment by lovvtide** [comment]
Ok, will fix. Thanks!
